app/rest/rest/pom.xml

```diff
@@ -209,6 +209,16 @@
       <artifactId>kubernetes-model</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Why?

Reply:
guess because of the maven dependency check croaking ? (and someone using okio directly in the code) But @zregvart probably knows best ;-)